services/ingest/tests/test_api_clusters.py

```
@@ -0,0 +1,496 @@
+@pytest.mark.asyncio
+async def test_create_cluster_requires_session(client: FastAPI) -> None:
+    from app.auth_middleware import get_current_user
+
+    client.dependency_overrides.pop(get_current_user, None)
+
+    async with AsyncClient(transport=ASGITransport(app=client), base_url="http://test") as ac:
+        resp = await ac.post("/api/clusters", json={"name": "prod-cluster"})
+    assert resp.status_code == 401
+
+
+def _integrity_error(sqlstate: str | None) -> "IntegrityError":
```

Ruff (0.16.3)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)